### PR TITLE
Lithuanian translation update

### PR DESCRIPTION
**Pull request by UrmasP**
Thx: Adga

### CODE REVIEW

File: po/lt.po [hunk 1]
```diff
@@ -163,10 +163,10 @@ msgid " (lower than any auto)"
 msgstr " (žemiau, nei bet koks auto)"
 
 msgid " extensions."
-msgstr " išplėtimai."
+msgstr "plėtiniai."
```
Comment (WanWizard) [inline po/lt.po]: There is a leading space missing on this line.

**Comment by UrmasP** [comment]
No idea, but I have to ask from the Lithuanian translator? I can correct, but its not my translation.

**Comment by UrmasP** [comment]
I think he can next time fix it, I will let know him! :-)

File: po/lt.po (hunk 1 above)
Comment (Adga52) [reply po/lt.po]: Error, I'm fixing it :)

Comment (UrmasP) [reply po/lt.po]: Adga, 
Thank you! Fixed one line.

**Pull request merged**